src/core/vms.rs

```diff
@@ -41,12 +44,30 @@ impl ResourceTrait for Vm {
         price_per_hour += (self.vm_ram_gb as f32) * self.price_license_per_ram_gb_per_hour;
         price_per_hour += self.price_license_per_vm_per_hour;
         price_per_hour += self.price_box_per_hour;
-        self.price_per_hour = Some(price_per_hour);
+        self.price_per_hour = match Some(price_per_hour) {
```

Comment:
you also want to update through `price_per_hour += ` as `self.price_per_month` is not using `self.price_per_hour` just below